srcpkgs/go-task/template

```diff
@@ -0,0 +1,31 @@
+# Template file for 'go-task'
+pkgname=go-task
+version=3.44.1
+revision=1
+build_style=go
+go_import_path=github.com/go-task/task/v3
+short_desc="Task runner / build tool that aims to be simple and easy"
+maintainer="Otavio Pliger <void@pliger.dev>"
+license="MIT"
+homepage="https://github.com/go-task/task"
+distfiles="https://github.com/go-task/task/archive/v${version}.tar.gz"
+checksum=d395eb802cca3f3f4b90e4bf504b6bc01f676f466d0bfb9e5045457bc085f516
+
+do_build() {
+	go build -o go-task ./cmd/task
```

Comment:
I would just rename the bianry in post_install, so you can acoid the `do_install` and the `do_build` function which allows the go build-style to pass all the flags to `go build`.

Reply:
I did a better read of the manual and code and got the response myself :)